app/src/main/java/org/mozilla/fenix/compose/MessageCard.kt

```diff
@@ -40,6 +41,7 @@ import org.mozilla.fenix.theme.FirefoxTheme
  * the title will not be shown.
  * @param buttonText An optional button text of the message card. If the button text is blank or null is provided,
```

Comment:
```suggestion
 * @param buttonText An optional button text of the message card. If the provided button text is blank or null,
```
Just realized my previous suggestion didn't read correctly. We also need to correct the description for `titleText`